functions-methods.js

```diff
@@ -20,7 +27,27 @@
 // typeOfEmail("novi.nlaapjesk@outlook.com") geeft geeft "Extern" <-- deze moet het ook doen!
 // typeOfEmail("a.wiersma@outlook.com") geeft "Extern"
 
+function typeOfEmail (emailadres) {
+
+    let email = emailadres.split('@')[1];
+
+    switch(email) {
+        case "outlook.com" :
+            email = "Extern";
+            break;
+        case "novi-education.nl" :
+            email =  "Student";
+            break;
+        case "novi.nl" :
+            email =  "Medewerker";
+            break;
+        default :
+            return "please fill in a valid input" ;
+    } return email;
```

Comment:
Je kunt ook direct hetgeen teruggeven. Dat scheelt wat regels code 😉 
```suggestion
    let email = emailadres.split('@')[1];

    switch(email) {
        case "outlook.com" :
            return "Extern";
        case "novi-education.nl" :
            return "Student";
        case "novi.nl" :
            return  "Medewerker";
        default :
            return "please fill in a valid input" ;
    };
```